.github/workflows/kind.yaml

```diff
@@ -85,14 +91,15 @@ jobs:
       - name: Set up external targets
         id: external_targets
         run: |
-          export worker2_ip=$(kubectl get nodes chart-testing-worker2 -o=jsonpath='{.status.addresses[?(@.type=="InternalIP")].address}')
-          export worker3_ip=$(kubectl get nodes chart-testing-worker3 -o=jsonpath='{.status.addresses[?(@.type=="InternalIP")].address}')
-          echo "worker2_ip=${worker2_ip}" >> $GITHUB_OUTPUT
-          echo "worker3_ip=${worker3_ip}" >> $GITHUB_OUTPUT
+          export worker6_ip=$(kubectl get nodes chart-testing-worker6 -o=jsonpath='{.status.addresses[?(@.type=="InternalIP")].address}')
+          export worker7_ip=$(kubectl get nodes chart-testing-worker7 -o=jsonpath='{.status.addresses[?(@.type=="InternalIP")].address}')
+          echo "worker6_ip=${worker6_ip}" >> $GITHUB_OUTPUT
+          echo "worker7_ip=${worker7_ip}" >> $GITHUB_OUTPUT
```

Comment:
I don't understand this change, there are several new nodes but then the test is changed to only use the last two nodes?

Reply:
[Two last nodes](https://github.com/cilium/cilium-cli/pull/2496/files#diff-94e083e5b848fc71129be9f77c0f2587aff5c3018835ada978a4fb75770e49f5R25-R33) have a special purpose: to host external (non-managed by Cilium) workloads.
So, I just increased the index for the external nodes after adding the new workers.

Reply:
Oh I see, I didn't look closely at the name of the step. Makes sense :+1: